esp-config/src/generate.rs

```diff
@@ -199,15 +229,56 @@ mod test {
         let mut v = Value::Bool(false);
 
         for input in TRUE_INPUTS {
-            v.parse(input).unwrap();
+            v.parse_in_place(input).unwrap();
             // no matter the input variant, the output format should be "true"
             assert_eq!(v.as_string(), "true");
         }
 
         for input in FALSE_INPUTS {
-            v.parse(input).unwrap();
+            v.parse_in_place(input).unwrap();
             // no matter the input variant, the output format should be "false"
             assert_eq!(v.as_string(), "false");
         }
     }
+
+    #[test]
+    fn env_override() {
```

Comment:
Please add a few cases that demonstrate fallback to default values, and error handling.

Reply:
Already on it, just wanted to see that the CI workflow was working :D